ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -4206,6 +4206,10 @@ private static void copyFiles(final HiveConf conf, final FileSystem destFs,
             }
           }
           files = fileStatuses.toArray(new FileStatus[files.length]);
+
+          if (HiveConf.getBoolVar(conf, HiveConf.ConfVars.HIVE_WRITE_ACID_VERSION_FILE)) {
+            AcidUtils.OrcAcidVersion.writeVersionFile(destf, destFs);
+          }
```

Comment:
It was a my first try to have it in the FSOperator which I have forgot to remove. After FSO we have like 3 moves which I wanted to avoid, so since we already have a Compactor specific change in copyFiles, I have decided to reuse it